README.md

```diff
@@ -1,3 +1,3 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Create files .env.test and a .env.development with the contents PGDATABASE=nc_news_test and PGDATABASE=nc_news respectively to connect the databases locally.
```

Comment:
Excellent. For this purpose (portfolio) it is fine to tell them what to set PGDATABASE to be in order to get it to work, but bear in mind, if this was for work purposes, you wouldn't want to publicly tell them the name of the databases set in the .env files.